.gitignore

```diff
@@ -3,3 +3,5 @@ layers/
 data/
 *.xml
 node_modules/
+.vagrant/
+Vagrantfile
```

Comment:
@pnorman It's mentioned in the commit, but it's ignored to avoid accidentally pushing a modified Vagrantfile unless it really is your intention, as you'd use it to configure the virtual machines number of CPU cores, memory, on/off for the hardware extensions for virtualization -- as well as changing which kosmtik plugins get installed during/by the Salt highstate / `vagrant provision`.